### DISCUSSION TITLE
Video Codec

### DISCUSSION THREAD

**Discussion by dsakura** [iRedScarf/onedrive-index#9] [Q&A] [answered]
Hello! Have a way to add more video support? Some videos run with black screen, just sound working. Thanks

**Comment by iRedScarf** [comment, marked as answer]
There are indeed various problems in watching videos online directly on the deployed OneDrive-Index web page, so it is recommended that you use the software below the video preview box to play, they include IINA, VLC, PotPlayer and nPlayer.

**Reply by manipinnaka1125** [reply]
mg
